### DISCUSSION TITLE
网易云新版更新后出现排版问题

### DISCUSSION THREAD

**Discussion by KaitouKidgmh** [amll-dev/applemusic-like-lyrics#227] [General]
网易云更新后顶栏扩大打开歌词界面顶栏不自动隐藏
![image](https://github.com/Steve-xmh/applemusic-like-lyrics/assets/124412473/3915ec4a-97cd-4e61-92da-0fed86536310)
以前会隐藏

**Comment by a0134** [comment]
使用StyleSnippet新增本地css并添加
.amll-lyric-page-open #page_pc_main_nav{
    transform: scale(0);
}

**Reply by KaitouKidgmh** [reply]
谢谢谢谢管用
